### ISSUE TITLE
NVDA cannot read MS Word highlighting without checking "Use UI Automation to access Microsoft Word document controls.

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#12101]
### Steps to reproduce:

1. Download the attached test.zip and unzip it.
2. Open the unzipped test.docx
3. Make sure "Use UI Automation to access Microsoft Word document controls" is unchecked in the NVDA advanced settings
4. Ensure that "Colors" is checked in the NVDA document formatting settings
5. Use the arrow keys to navigate through this document.
6. Check "Use UI Automation to access Microsoft Word document controls when available" in the NVDA advanced settings.
7. Use the arrow keys to navigate through this document.

### Actual behavior:
1. Execute step (5) NVDA will read out "I am a student."
2. Execute step (7) NVDA will read out "I bright yellow background am white background a grey background student white background."

### Expected behavior:
Both step (5) and step (7) should be read aloud as """I bright yellow background am white background a grey background student white background."

#### NVDA installed/portable/running from source:
installed andportable
#### NVDA version:
NVDA2020.3 andNVDA2020.4
#### Windows version:
Windows 10 20H2 (64-bit) build 19042.844

#### Name and version of other software in use when reproducing the issue:
Microsoft Word for Microsoft 365 MSO (16.0.13628.20318) 64-bit 

#### Other information about your system:
none

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes

**Comment by cary-rowen** [comment]
Here is the document for testing, please download and try to unzip it.
[test.zip](https://github.com/nvaccess/nvda/files/6047289/test.zip)

**Comment by cary-rowen** [comment]
@feerrenrut @leonardder Could you please test this issue?
Thanks!

**Comment by dingpengyu** [comment]
cc @michaelDCurran

**Comment by cary-rowen** [comment]
Is anyone following this issue?
